backend/services/inventory/api/http/api_inventory.go

```diff
@@ -50,19 +50,20 @@ const (
 	uriAttributes       = "/attributes"
 	uriDeviceAttributes = "/device/attributes"
 
-	apiUrlInternalV1         = "/api/internal/v1/inventory"
-	uriInternalAlive         = "/alive"
-	uriInternalHealth        = "/health"
-	uriInternalTenants       = "/tenants"
-	uriInternalDevices       = "/tenants/:tenant_id/devices"
-	urlInternalDevicesStatus = "/tenants/:tenant_id/devices/status/:status"
-	uriInternalDeviceDetails = "/tenants/:tenant_id/devices/:device_id"
-	uriInternalDeviceGroups  = "/tenants/:tenant_id/devices/:device_id/groups"
-	urlInternalAttributes    = "/tenants/:tenant_id/device/:device_id/attribute/scope/:scope"
-	urlInternalReindex       = "/tenants/:tenant_id/devices/:device_id/reindex"
-	apiUrlManagementV2       = "/api/management/v2/inventory"
-	urlFiltersAttributes     = "/filters/attributes"
-	urlFiltersSearch         = "/filters/search"
+	apiUrlInternalV1             = "/api/internal/v1/inventory"
+	uriInternalAlive             = "/alive"
+	uriInternalHealth            = "/health"
+	uriInternalTenants           = "/tenants"
+	uriInternalDevices           = "/tenants/:tenant_id/devices"
+	urlInternalDevicesStatus     = "/tenants/:tenant_id/devices/status/:status"
+	uriInternalDeviceDetails     = "/tenants/:tenant_id/devices/:device_id"
+	uriInternalDeviceGroups      = "/tenants/:tenant_id/devices/:device_id/groups"
+	urlInternalAttributes        = "/tenants/:tenant_id/device/:device_id/attribute/scope/:scope"
+	urlInternalAttributesNoScope = "/tenants/:tenant_id/device/:device_id/attribute"
```

Comment:
The damage is already done to the path right above, but I think we should fix the grammar on the new one.
```suggestion
	urlInternalAttributesNoScope = "/tenants/:tenant_id/device/:device_id/attributes"
```

Reply:
sure. fixed.